draft-private-access-tokens.md

```diff
@@ -780,28 +780,36 @@ certificate pinning, to mitigate the risk of channel compromise; see
 {{sec-considerations}} for additional about this channel.
 
 Issuance begins by Clients hashing the TokenChallenge to produce a token input
-as message = SHA256(challenge), and then blinding message as follows:
+as `message = SHA256(challenge)`, and then blinding `message` as follows:
 
 ~~~
 blinded_req, blind_inv = rsabssa_blind(ORIGIN_TOKEN_KEY, message)
 ~~~
 
-The Client MUST use a randomized variant of RSABSSA in producing this signature with
-a salt length of at least 32 bytes.
+The Client MUST use a randomized variant of RSABSSA in producing this signature
+with a salt length of at least 32 bytes.
+
+The Client then uses CLIENT_KEY to generate its index request `index_request`
+and blind `index_blind` as described in {{client-stable-mapping}}.
+
+The Client then encrypts the origin name using ISSUER_KEY, producing
+`issuer_key_id` and `encrypted_origin_name` as described in {{encrypt-origin}}.
 
-The Client uses CLIENT_KEY and CLIENT_SECRET to generate proof of its request as
-described in {{client-stable-mapping}}, yielding output client_origin_index_blind and
-client_origin_index_request. The Client then constructs a Private Access Token
-request using client_origin_index_request, blinded_req, and origin information.
+Finally, the Client uses CLIENT_SECRET to produce `request_proof`
+as described in {{index-proof}}.
```

Comment:
n00b question: what is the purpose of this proof?

Reply:
The new formulation should be much simpler - please have a look!